package/template-cartesian-graph/package

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(template-cartesian-graph)
+pkgdesc="A Cartesian graph template for the reMarkable tablet"
+url=https://github.com/PeterGrace/template-cartesian-graph
+pkgver=1.0.0-4
+timestamp=2021-01-14T17:46Z
+section=utils
+maintainer="Peter Grace <pete.grace@gmail.com>"
+license=MIT
+depends=(templatectl)
+
+image=qt:v1.2.2
+source=(https://github.com/PeterGrace/template-cartesian-graph/archive/v1.0.0.zip)
```

Comment:
You could use `$pkgver` here to avoid duplicating the version number.
```suggestion
source=("https://github.com/PeterGrace/template-cartesian-graph/archive/v${pkgver%-*}.zip")
```